command.go

```diff
@@ -104,23 +103,7 @@ func (c *Command) Call(req *Request, re ResponseEmitter, env Environment) (err e
 		}
 	}
 
-	defer func() {
```

Comment:
Dead code. I'm not a fan of recovering from panics anyways (usually). We have no guarantees about the current state at that point.